packages/wasm-terminal/src/command/callback-command.ts

```diff
@@ -1,9 +1,12 @@
 import Command from "./command";
 import CommandOptions from "./command-options";
+import { WasmFs } from "@wasmer/wasmfs";
+import { Duplex } from "stream";
```

Comment:
Are we using Duplex here? Or was this left by accident? 🤔

Reply:
By accident! Removing it